### PR TITLE
Added functionality and improvised detection logic

### PR DESCRIPTION
**Pull request by dpshekhawat**
This PR includes - 

- timestamp & detected objects stored in json file _(metadata)_
- if objects are detected, then only files are written into the folder _(improvised logic)_
- folder creation moved to export _(for data sync [reference](https://project-flotta.io/documentation/v0_2_0/operations/data_synchronization.html#bi-directional-configuration))_
- code reformatting and cleaning.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -25,61 +25,84 @@
...
+                self.frame,
+                (x, y),
+                (x + w, y + h),
+                color=(0, 200, 50),
+                thickness=2,
+            )
+
+        return detected
+
+def jsonData(title, detected, folder):
```
Comment (pkliczewski) [inline main.py]: what do you think about `write_metadata` instead? `jsonData` is not descriptive

File: main.py [hunk 2]
```diff
@@ -25,61 +25,84 @@
...
-            cv2.imwrite(os.path.join(folder, filename), frame)
-
+            classifier = objDetection(frame, thres)
+            detected = classifier.objDetect()
+            
+            timestamp = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+            filename = f"{timestamp}.jpeg"
+
+            if len(detected)!=0:
+                # Saving data to specified folder.
```
Comment (pkliczewski) [inline main.py]: What do you think about moving file writing logic to separate function? We could have it there timestamp and both line 101 and 102.

File: main.py [hunk 3]
```diff
@@ -25,61 +25,84 @@
...
-            thickness=2,
-        )
-
-        cv2.rectangle(
-            frame,
-            (x, y),
-            (x + w, y + h),
-            color=(0, 200, 50),
-            thickness=2,
+class objDetection():
```
Comment (pkliczewski) [inline main.py]: the script is small but if you like to have a class so let's instantiate it. We can have a main function where we create the object and call funtions.

File: main.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 import os
```
Comment (pkliczewski) [inline main.py:4]: Please follow https://pythonguides.com/python-naming-conventions/

File: main.py (hunk 3 above)
Comment (dpshekhawat) [reply main.py]: I thought of adding few methods specific to the detection but that's not the case right now so I think we can introduce class in further updates if required. thank you!

**Review by pkliczewski — APPROVED** [review]

**Pull request merged**